### PR TITLE
github CI building el8stream images

### PR DESCRIPTION
**Pull request by michalskrivanek**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        known_hosts: ${{ secrets.KNOWN_HOSTS_FOR_RESOURCES_OVIRT_ORG }}
+        source: rpmbuild/RPMS/x86_64/*.rpm
+        target: github-ci/ost-images
+        cleanup: yes
+        createrepo: yes
+        # keep 10 last rpms (not necessarily complete set of images) + repodata
+        keep_files_count: 11
+
+    - name: Clean up entire workdir
+      if: always()
```
Comment (tinez) [inline .github/workflows/build.yml:55]: That construct provides something like a `finally` block?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        required: false
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  check:
+    if: |
+      github.event.issue.pull_request &&
+      startsWith(github.event.comment.body, '/ost') &&
```
Comment (tinez) [inline .github/workflows/build.yml:16]: We probably want to use a different verb than `/ost`

**Review by tinez — COMMENTED** [review]
Shouldn't we have the `self-hosted` mentioned somewhere?

File: .github/workflows/build.yml (hunk 1 above)
Comment (michalskrivanek) [reply .github/workflows/build.yml:55]: yes
it's jsut that we're a bit short on space if all the projects left their images and squashfs and whatnot...

File: .github/workflows/build.yml (hunk 2 above)
Comment (michalskrivanek) [reply .github/workflows/build.yml:16]: eventually sure

**Pull request merged**